src/SpotifyApi.NetCore/FollowApi.cs

```diff
@@ -143,5 +143,47 @@ public FollowApi(HttpClient httpClient) : base(httpClient)
             return await GetModel<T>(builder.Uri, accessToken);
         }
         #endregion
+
+        #region FollowArtistsOrUsers
+        /// <summary>
+        /// Add the current user as a follower of one or more artists Spotify IDs.
+        /// </summary>
+        /// <param name="artistIds">Required. A comma-separated list of the artist Spotify IDs.</param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/follow-artists-users/
+        /// </remarks>
+        public async Task FollowArtists(
+            string[] artistIds,
+            string accessToken = null
+            )
+        {
+            if (artistIds?.Length < 1 && artistIds.Length > 50) throw new ArgumentNullException("artistIds");
+
+            var builder = new UriBuilder($"{BaseUrl}/me/following");
+            builder.AppendToQuery("type", "artist");
+            builder.AppendToQueryAsCsv("ids", artistIds);
+            await PutModel(builder.Uri, accessToken);
```

Comment:
Just call `Put` here, e.g.

    await Put(builder.Uri, null, accessToken);

If you pull from origin master I have added a Uri overload for this helper 🙂